requirements.txt

```diff
@@ -12,7 +12,6 @@
 # Required packages.
 # -------------------------------------------------------------------
 # install:required
-setuptools<66.0.0
```

Comment:
This does not look right (but I might be wrong). Why don't we need setup tools as a requirement?

I had expected that only the version limit to be removed.

Reply:
https://github.com/pymodbus-dev/pymodbus/pull/1373

See the original pin here.  `setuptools` was never required, presumably since it's almost always installed with Python itself.

https://stackoverflow.com/questions/64811350/is-setuptools-always-installed-in-python-by-default

Reply:
`setuptools` I'd an internal requirement to make sure a target machine has required version satisfied even though the Python installation comes with it. Not everyone updates the default packages. It's better to have it pinned.